src/App.vue

```diff
@@ -10,7 +10,8 @@
     <!-- Twitter -->
     <meta name="twitter:title" content="ROAR Web Query" />
     <meta name="twitter:description" content="A web-based tool to query ROAR assessment data!" />
-  </AppHead>
+  </Head>
```

Comment:
Thanks for catching this, I didn't realize that our `AppHead` component was just using the default `Head` component from @vueuse